patches/snappy/snappy-1.1.9_gcc_inline.patch

```diff
@@ -0,0 +1,67 @@
+From 0c716d435abe65250100c2caea0e5126ac4e14bd Mon Sep 17 00:00:00 2001
```

Comment:
This file does not match our usual patch naming convention.

Reply:
The patch was taken as-is from gentoo, so I kept the name intact as well.